www/core/teststs/extension_sts.xml

```diff
@@ -57,4 +57,23 @@
 		<targetplatform name="joomla" version="3.6"/>
 		<php_minimum>5.3.10</php_minimum>
 	</update>
+	<update>
+		<name>Joomla! 3.6</name>
+		<description>Joomla! 3.6 CMS</description>
+		<element>joomla</element>
+		<type>file</type>
+		<version>3.6.3-rc1</version>
+		<infourl title="Joomla!">https://www.joomla.org/announcements/release-news/5667-joomla-3-6-2-released.htmlg</infourl>
+		<downloads>
+			<downloadurl type="full" format="zip">https://github.com/joomla/joomla-cms/releases/download/3.6.3-rc1/Joomla_3.6.3-rc1-Release_Candidate-Update_Package.zip</downloadurl>
+		</downloads>
+		<tags>
+			<tag>stable</tag>
+		</tags>
+		<maintainer>Joomla! PLT</maintainer>
+		<maintainerurl>http://www.joomla.org</maintainerurl>
+		<section>STS</section>
+		<targetplatform name="joomla" version="3.6" max_dev_level="0"/>
```

Comment:
And this one should have a download url of Joomla_3.6.2_to_3.6.3-rc1-Release_Candidate-Patch_Package.zip and have a min_dev_level of 2